src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -176,15 +178,17 @@ internal class WamBroker : IBroker
                         _logger,
                         isInteractive: true);
                 }
-                else
-                {
-                    if (IsAadOnlyAuthority(authenticationRequestParameters.Authority))
-                    {
-                        return await AcquireInteractiveWithAadBrowserAsync(
-                            authenticationRequestParameters,
-                            acquireTokenInteractiveParameters.Prompt).ConfigureAwait(false);
-                    }
-                }
+            }
+
+            // no account information available, need an account picker
```

Comment:
In the issue you mention the "WAM AAD plugin interactive experience" Is this an account picker with no MSA accounts or just the standard sign in experience?

Reply:
I updated the issue with some screenshots to make it clear 

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2289